src/Components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+import "./Sidebar.css";
+
+export const Sidebar = () => {
+  const [click, setClick] = useState(true);
+  return (
+    <aside>
+      <div className="sidebar">
+        <ul>
+          <li
+            onClick={() => setClick(true)}
+            className={`${click ? "selected" : ""}`}
```

Comment:
you can remove outer parenthesis just this `${click ? "selected" : ""}` will work

Reply:
thanks for the suggestion. this state is temporary so it will be removed